packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties

```diff
@@ -2,6 +2,7 @@ breadcrumbs.home.text = Home
 breadcrumbs.label.text = Breadcrumbs
 close.text = close
```

Comment:
Why is `close` all lower case? Shouldn't it be capitalized `Close`?

Reply:
yeah I feel like it should be. is the change ok to leave in if I change the key in the properties file?

Reply:
I'd argue it's ok. The letter casing issue isn't a breaking translation change

Reply:
Yes, it is not a breaking translation change as you said, but I think if it is used only for the tooltip, and the other tooltips are capitalized, we can be consistent. 
Stephanie, can you check if this key is used somewhere else?

Reply:
it is also used in Dialog.tsx and Toast.tsx, but as aria labels. is it ok to change the key to upper case?

Reply:
I would use `"Close"`.

Reply:
Yeah, let's change it to capitalized